esp/services/esdl_svc_engine/esdl_binding.hpp

```diff
@@ -109,6 +109,7 @@ class EsdlServiceImpl : public CInterface, implements IEspService
     StringBuffer                m_serviceNameSpaceBase;
     StringAttr                  m_namespaceScheme;
     bool                        m_usesURLNameSpace;
+    bool                        m_returnSchemaLocationOnOK;
```

Comment:
For new values we should really be initializing them here.. our code will improve that way over time.  Can then remove the setting in init() assuming init() is only called once.  If not, you can leave it there as well.